### ISSUE TITLE
Midpoint Review - Design Feedback - 10-10 Health Apps team, 10-10EZR Health Benefits Update form

### ISSUE THREAD

**Issue by allison0034** [department-of-veterans-affairs/va.gov-team#68457]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- Green sign in on intro page?

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- Matt Dingee: Use new v3 components when you can.
- **Must**: Review the [guidelines for additional info component](https://design.va.gov/components/additional-info#when-to-consider-something-else). Make sure you do not have too much content/multiple paragraphs.

**Comment by jessicastump** [comment]
For v3 components - yes, development is using whatever was available in the forms library at the time and we plan to address the progress bar and other components that were not included with the mvp launch in an upcoming release. 

For the additional information component - yes, the 1010 forms do contain a few! 
We reviewed and realize the ones in the Veteran information section seem to align with the guidelines. The content in the add info components within the household financial information section were all recently worked on and heavily reviewed on the EZ form, the same content was used here. Most of these align with guidelines too - maybe a lengthy sentence or two followed by a list of examples. These were intended to provided helpful examples in context to the input field and these were complicated questions that required additional context and explanation. 

However, the additional info content used on the private insurance page is lengthly and could potentially be out of component guidelines. This content is formatted with this component today on the EZ and is live, but has not been reviewed specifically for some time. If we move the content out of the additional info component for this one page/question, would that meet the MUST requirements here?

**Comment by allison0034** [comment]
@jessicastump yes, I do not see any other additional info components that have multiple paragraphs.

**Comment by jessicastump** [comment]
Thank you @allison0034. To clarify, is that change a must still for only that one use of the additional info component on the private insurance page or could that be a should item for mvp?

**Comment by allison0034** [comment]
@jessicastump I did sync with Matt on this, is there anyway you can shorten the content and keep it in the additional info component? Moving it to an accordion that follows the form fields may be an issues with screenreader users. Screenreader users may not find it until they have already answered the question.

**Comment by jessicastump** [comment]
Thank you @allison0034!! 

We took a look at this again - and we have been working closely with Aliyah on the CAIA team for all of these updates. We workshoped this via the HE UX team then ran by Aliyah real quick too. 

We have an [updated screen here](https://www.sketch.com/s/912cab8e-d234-44dd-be1f-2bedb3f50b22/v/ey2nmj/a/vRr8rR3) - no official ticket yet - that we worked on to meet the additional info component concerns. 

I would like to circle back to if this is a must for mvp and if it could be a should. We are planning on looking at some insurance content on the intro page as well and plan to release an update very quickly after mvp and could include these updates then. Is updating the insurance page still a must?

Please let me know and thank you for helping us with this quick turnaround :)

**Comment by allison0034** [comment]
It is a must, but looking at your most recent screen above I think it fits the guidance now!
